loggingFpdb.py

```diff
@@ -0,0 +1,121 @@
+import os
+import re
+import logging
+import colorlog
+import inspect
+
+
+# Set default logging and debugging modes
+def set_default_logging():
+    """Sets logging to show only warnings and errors."""
+    logging.getLogger().setLevel(logging.WARNING)
+
+
+def enable_debug_logging():
+    """Sets logging to show all levels, including info and debug."""
+    logging.getLogger().setLevel(logging.DEBUG)
+
+
+class FpdbLogFormatter(colorlog.ColoredFormatter):
+    """Formateur personnalisé pour les logs FPDB"""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.var_color = "\033[36m"  # Cyan pour les variables
+        self.reset_color = "\033[0m"
+
+    def format(self, record):
+        if hasattr(record, "msg"):
+            record.msg = self._colorize_variables(record.msg)
+        return super().format(record)
+
+    def _colorize_variables(self, message):
+        if not isinstance(message, str):
+            return message
+        pattern = r"'([^']*)'|\"([^\"]*)\""
```

Comment:
You should always explain a regex goal. A bit like in poker you should verbalize the amount of your raise when you raise; here you should add a comment to quickly explain "Capture quoted text (simples or doubles)". This took me some time to get the idea (regex is not the easiest and quicker langage to read).
This is even more important when the function name doesn't correspond to the code. A function named "colorize variables" should colorize all variables types (int, lists, dicts, sets ...) not only strings.